src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -2350,6 +2355,34 @@ mongoc_uri_get_ssl (const mongoc_uri_t *uri) /* IN */
    return mongoc_uri_get_tls (uri);
 }
 
+const char *
+mongoc_uri_get_server_monitoring_mode (const mongoc_uri_t *uri)
+{
+   BSON_ASSERT (uri);
```

Comment:
```suggestion
   BSON_ASSERT_PARAM (uri);
```

Suggest using `BSON_ASSERT_PARAM` for a more helpful message on failure.

With `BSON_ASSERT`, the assertion error message is:
```
precondition failed: uri
```
With `BSON_ASSERT_PARAM`
```
The parameter: uri, in function mongoc_uri_get_server_monitoring_mode, cannot be NULL
```

Suggestion also applies to `mongoc_uri_set_server_monitoring_mode`.